setup.py

```diff
@@ -5,6 +5,19 @@
     version="0.1.0",
     author="Andreas Doerr",
     author_email="andreasdoerr@gmx.net",
+    license='MIT',
     description="Tool to automatically query impfterminservice.de",
     packages=setuptools.find_packages(),
+    python_requires='>=3.6',
+    classifiers=[
+        'License :: OSI Approved :: MIT License',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
```

Comment:
How's this with respect to `python_requires='>=3.6'`. Not sure which min. Python version would really be necessary.

Reply:
oops. wait a sec...

Reply:
fixed.

3.6 is the minimum version supported by python development.

also, it is nice as a minimum version as it supports f-strings and some other nice features not present in older versions.